definitions/npm/validate-currency-code_v0.x.x/flow_v0.104.x-/test_validate-currency-code_v0.x.x.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import isCurrencyCode from 'validate-currency-code';
+
+
+const value: boolean = isCurrencyCode('USD')
+
+// $FlowExpectedError if string is not passed into validator
```

Comment:
You'll need to add error codes to suppression comments. See failing CI tests.

E.g.: `$FlowExpectedError[incompatible-call]`